src/quo2/components/numbered_keyboard/keyboard_key/view.cljs

```diff
@@ -0,0 +1,30 @@
+(ns quo2.components.numbered-keyboard.keyboard-key.view
+  (:require [react-native.core :as rn]
+            [quo2.theme :as quo.theme]
+            [quo2.components.markdown.text :as text]
+            [quo2.components.icon :as icons]
+            [quo2.components.numbered-keyboard.keyboard-key.style :as style]
+            [reagent.core :as reagent]))
+
+(defn- keyboard-key-internal
+  []
+  (let [pressed? (reagent/atom false)]
+    (fn
+      [{:keys [disabled? theme blur? on-press type]} label]
+      (let [label-color      (style/get-label-color disabled? theme blur?)
+            background-color (style/toggle-background-color @pressed? blur? theme)]
+        [rn/touchable-without-feedback
```

Comment:
@J-Son89 Jamie suggested to use `rn/touchable-without-feedback`. Please let me know which one should I go with? @ilmotta

Reply:
pressable is good, as long as we set the styling to our own designs and not based on the react-native components internal stylings 👍

Reply:
Done

Reply:
> If you're looking for a more extensive and future-proof way to handle touch-based input, check out the [Pressable](https://reactnative.dev/docs/pressable) API.
>
> -- https://reactnative.dev/docs/touchablewithoutfeedback

@mmilad75, @J-Son89, all RN Touchable* have this yellow warning right at the top. It seems prudent that we define a guideline for using `rn/pressable` so that we can gradually phase out Touchable* from status-mobile.

cc @OmarBasem, @flexsurfer, @Parveshdhull, @smohamedjavid what do you guys think? Worth of a quick PR to add to the guidelines?

Reply:
@ilmotta Yeah I think we should start using Pressable. Although other touchable components are still maintained, I think it is better to start migrating sooner. Also, would unify the touchable components instead of having 3 different components.

Reply:
@OmarBasem I replaced it with `Pressable`